src/Protocol/Http/Authenticator/JwtAuthenticator.php

```diff
@@ -30,24 +30,50 @@ class JwtAuthenticator implements AuthenticatorInterface
      */
     private $jwt;
 
+    /**
+     * @var string
+     */
+    private $jws;
+
+    /**
+     * @var \DateInterval
+     */
+    private $jwsLifetime;
+
+    /**
+     * @var \DateTime
+     */
+    private $jwsValidTo;
+
     /**
      * Constructor.
      *
-     * @param JwtInterface $jwt
+     * @param JwtInterface  $jwt
+     * @param \DateInterval $jwsLifetime
+     * @throws \InvalidArgumentException
      */
-    public function __construct(JwtInterface $jwt)
+    public function __construct(JwtInterface $jwt, \DateInterval $jwsLifetime = null)
     {
         $this->jwt = $jwt;
+        if ($jwsLifetime && $jwsLifetime->invert) {
```

Comment:
Please add the blank line before `if` condition. This is code style for easy reading the business logic of method.